<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoffeeEasterEggWidget</class>
 <widget class="QWidget" name="CoffeeEasterEggWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lbScreenControl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Pour bien bosser, il faut du café!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbCoffee">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnCoffeeMaker">
     <property name="text">
      <string>Faire le café !</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/icons/img/coffee_ready.png</normaloff>:/icons/img/coffee_ready.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
